ote/src/cljs/ote/ui/buttons.cljs

```diff
@@ -45,3 +46,10 @@
    (common/linkify url
                    [ui/flat-button {:label label :primary true}]
                    {:target "_blank"})])
+
+(defn icon-button
+  [opts label]
+  [:button.unstyled-button (merge
+             opts
```

Comment:
I wonder, should `opts` be last in `merge` to allow overwriting the default style from stylefy call?